config/config.go

```diff
@@ -828,6 +836,47 @@ func (c *KubernetesSDConfig) UnmarshalYAML(unmarshal func(interface{}) error) er
 	return nil
 }
 
+// GCESDConfig is the configuration for GCE based service discovery.
+type GCESDConfig struct {
+	// Project: The Google Cloud Project ID
+	Project string `yaml:"project"`
+
+	// Zone: The zone of the scrape targets.
+	// If you need to configure multiple zones use multiple gce_sd_configs
+	Zone string `yaml:"zone"`
+
+	// Filter: Can be used optionally to filter the instance list by other criteria.
+	// Syntax of this filter string is described here in the filter query parameter section:
+	// https://cloud.google.com/compute/docs/reference/latest/instances/list
+	Filter string `yaml:"filter,omitempty"`
```

Comment:
How would you like me to expose the tags? Since there can be several tags on one instance and there is no real key-value pair (the tag is the value), how should I expose them?

Should they be enumerated like:
```
__meta_gce_tag_0: my-tag
__meta_gce_tag_1: my-other-tag
```
Of course this would need to be sorted, because we can't be sure that the api answer has always the same order of tags.
This way could also make relabelling harder, especially if tags change.

Or we could use the value twice like:
```
__meta_gce_tag_my-tag: my-tag
__meta_gce_tag_my-other-tag: my-other-tag
```

Any other ideas or suggestions how to do this in a reasonable way?